testsuite/tests/typing-layouts-bits16/unboxed_bits16s.ml

```diff
@@ -1,17 +1,17 @@
 (* TEST
- include stdlib_beta;
+ include stdlib_stable;
  flambda2;
  {
-   flags = "-extension layouts_alpha -extension small_numbers_beta";
+   flags = "-extension layouts_alpha -extension small_numbers";
    native;
  }{
-   flags = "-extension layouts_alpha -extension small_numbers_beta";
+   flags = "-extension layouts_alpha -extension small_numbers";
    bytecode;
  }{
-   flags = "-extension layouts_beta -extension small_numbers_beta";
+   flags = "-extension layouts_beta -extension small_numbers";
    native;
  }{
-   flags = "-extension layouts_beta -extension small_numbers_beta";
+   flags = "-extension layouts_beta -extension small_numbers";
```

Comment:
I think the idea of the in tests like this is to have one stanza for stable and one beta (and also the combinations of native/bytecode), so we are checking the extension level doesn't change the behavior.

(And like the other tests, I think the layouts part should also be changing)